### PR TITLE
Fix sparse_categorical_crossentropy argument and add value test for it  (#7513)

### PR DESCRIPTION
**Pull request by nzw0301**
If Keras needs to add value checking test for other loss functions, I will update this PR for them.
What should I do?

### CODE REVIEW

**Comment by nzw0301** [comment]
CNTK's case:

```python
y_pred = K.variable(np.array([[0.3, 0.6, 0.1],
                                                 [0.1, 0.2, 0.7]]))
y_true = K.variable(np.array([1, 2]))
expected_loss = - (np.log(0.6) + np.log(0.7)) / 2
loss = K.eval(losses.sparse_categorical_crossentropy(y_true, y_pred))
```

- `np.mean(loss)`: ` 5.6413336`
- `expected_loss`:  `0.43375028385236158`
